tests/test_free_energy_rabfe.py

```diff
@@ -23,3 +31,45 @@ def test_rabfe_result_to_from_log():
 
     reconstructed = RABFEResult.from_log(first_line)
     assert result == reconstructed
+
+def test_setting_up_restraints_using_distance():
+    seed = 814
+    smi_a = "CCCONNN"
+    smi_b = "CCCNNN"
+    mol_a = Chem.MolFromSmiles(smi_a)
+    mol_a = Chem.AddHs(mol_a)
+    mol_b = Chem.MolFromSmiles(smi_b)
+    mol_b = Chem.AddHs(mol_b)
+    for mol in [mol_a, mol_b]:
+        AllChem.EmbedMolecule(mol, randomSeed=seed)
+
+    core = setup_relative_restraints_by_distance(mol_a, mol_b)
```

Comment:
we should add a test asserting that all distances from this core are less than cutoff

Reply:
(and probably a good idea to test for more cutoffs)